src/Infrastructure/HConfig/interface/ESMF_HConfig.F90

```diff
@@ -13589,6 +13589,102 @@ subroutine ESMF_HConfigSetMapValStringSeq(hconfig, content, keywordEnforcer, &
 !------------------------------------------------------------------------------
 
 
+! -------------------------- ESMF-public method -------------------------------
+#undef  ESMF_METHOD
+#define ESMF_METHOD "ESMF_HConfigValidateMapKeys()"
+!BOP
+! !IROUTINE: ESMF_HConfigValidateMapKeys - Validate map keys against list of vocabulary
+
+! !INTERFACE:
+  function ESMF_HConfigValidateMapKeys(hconfig, vocabulary, keywordEnforcer, &
+    badKey, rc)
+!
+! !RETURN VALUE:
+    logical :: ESMF_HConfigValidateMapKeys
+!
+! !ARGUMENTS:
+    type(ESMF_HConfig),        intent(in)            :: hconfig
+    character(len=*),          intent(in)            :: vocabulary(:)
+type(ESMF_KeywordEnforcer), optional:: keywordEnforcer ! must use keywords below
```

Comment:
Are you wondering if line 13608 should have some whitespace to line up? It does not, because the ProTex processor picks it up without whitespace, and inserts the actual language for optional args requiring keywords.